### DISCUSSION TITLE
Fullstack.Net - Excluindo Categoria - Relacionamento 1:N

### DISCUSSION THREAD

**Discussion by atilionogueira** [balta-io/forum#694] [Tira Dúvidas] [answered]
Pergunta??
Se no lançamento tiver supermercado, não poderia no modulo Categoria conseguir deletar o registro? Correto?? Mas o app controle financeiro deixa.. como resolver isto???
Como Categoria tem o relacionamento (Categoria)1 :N(Transação), correto??

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado: https://github.com/balta-io/forum/discussions/1#discussioncomment-10942169
